src/project.jl

```diff
@@ -170,12 +170,11 @@ project_key_order(key::String) =
 
 write_project(project::Project, project_file::AbstractString) =
     write_project(destructure(project), project_file)
-write_project(project::Project, io::IO) =
-    write_project(destructure(project), io)
```

Comment:
So I guess we never call `write_project(::Project, ::IO)`?

Reply:
No :)